QuickOSM/core/process.py

```diff
@@ -34,15 +35,15 @@
 
 
 def open_file(
-        dialog=None,
-        osm_file=None,
+        dialog: QDialog = None,
+        osm_file: str = None,
         output_geom_types=None,
         white_list_column=None,
-        layer_name="OsmFile",
+        layer_name: str = "OsmFile",
         config_outputs=None,
```

Comment:
You weren't sure for these 3 parameters ?
* output_geom_types, from my memories it's a list
* white_list_column is a dict I think
* I'm not sure for config_outputs, can you check the code